### PR TITLE
feat(Picker): Add a point picker

### PR DESCRIPTION
**Pull request by laurennlam**
Add the point picker to pick 3D point from mouse clicking.

### CODE REVIEW

File: Sources/Rendering/Core/AbstractPicker/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+// ----------------------------------------------------------------------------
+
+export function extend(publicAPI, model, initialValues = {}) {
+  Object.assign(model, DEFAULT_VALUES, initialValues);
+
+  // Build VTK API
+  macro.obj(publicAPI, model);
+
+  macro.get(publicAPI, model, ['renderer', 'pickList']);
+  macro.getArray(publicAPI, model, ['selectionPoint', 'pickPosition'], 3);
```
Comment (jourdain) [inline Sources/Rendering/Core/AbstractPicker/index.js]: I don't think the getArray macro needs to have the size. Only the set or set/get will need it.

**Comment by jourdain** [comment]
I let @martinken review it.

**Comment by martinken** [comment]
I love it. +2

**Comment by jourdain** [comment]
@laurennlam if you can fix the `macro.getArray` call that could be nice. 
That can be done in a chore(...)

FYI
```
export function getArray(publicAPI, model, fieldNames)
// ----------------------------------------------------------------------------
// setXXX: add setter for object of type array
// if 'defaultVal' is supplied, shorter arrays will be padded to 'size' with 'defaultVal'
// ----------------------------------------------------------------------------
export function setArray(publicAPI, model, fieldNames, size, defaultVal = undefined)
export function setGetArray(publicAPI, model, fieldNames, size, defaultVal = undefined)
```

**Comment by laurennlam** [comment]
@jourdain What do you mean by fixing the` macro.getArray` ?

**Comment by jourdain** [comment]
No, I mean calling the function with the proper set of arguments

`macro.getArray(publicAPI, model, ['selectionPoint', 'pickPosition'], 3);` vs `macro.getArray(publicAPI, model, ['selectionPoint', 'pickPosition']);`

**Pull request merged**